src/adapter.js

```diff
@@ -319,11 +321,24 @@ var createSpecFilter = function (config, jasmineEnv) {
 function createStartFn (karma, jasmineEnv) {
   // This function will be assigned to `window.__karma__.start`:
   return function () {
+    var clientConfig = karma.config || {}
+    var jasmineConfig = clientConfig.jasmine || {}
+
     jasmineEnv = jasmineEnv || window.jasmine.getEnv()
 
+    setOption(jasmineConfig.stopOnFailure, jasmineEnv.throwOnExpectationFailure)
+    setOption(jasmineConfig.seed, jasmineEnv.seed)
+    setOption(jasmineConfig.random, jasmineEnv.randomizeTests)
+
     jasmineEnv.addReporter(new KarmaReporter(karma, jasmineEnv))
     jasmineEnv.execute()
   }
+
+  function setOption (option, set) {
+    if (option !== undefined) {
```

Comment:
Better to use `!= null`, so it checks for both `undefined` and `null`. Or you could use `lodash.merge` to just merge the config objects.